packages/@uppy/status-bar/src/StatusBar.jsx

```diff
@@ -130,7 +157,6 @@ export default class StatusBar extends UIPlugin {
       newFiles,
       startedFiles,
       completeFiles,
-      inProgressNotPausedFiles,
```

Comment:
Shouldn't paused files be taken into account too when computing ETA, since they are also used for `totalBytes` and `totalUploadedBytes`?  If I'm wrong about that, this could be reverted. 
The reason I decided to go for it is because it saves one full iteration over the files to compute the sizes again.

Reply:
We discussed this internally, and decided it was OK the way you set it up. We noticed that when the full UI is paused, then un-paused a few minutes later, the ETA is much too large, probably because the computation is based on the upload start, which is artificially too far in the past. It would be nice if we could update the time of origin when the user un-pause, but it's not the end of the world as is.